releaser-core/src/main/java/releaser/internal/git/GitRepo.java

```diff
@@ -184,6 +184,21 @@ void commit(String message) {
 		}
 	}
 
+	void checkoutTag(String tagName) {
+		try (Git git = this.gitFactory.open(file(this.basedir))) {
+			Optional<ObjectId> tagId = findTagIdByName(tagName, false);
+			if (tagId.isPresent()) {
```

Comment:
I think you can change this to `map` and then `orElseThrow` or sth like that. No need to do that though